app/api/users/specs/users.spec.js

```diff
@@ -119,6 +121,23 @@ describe('Users', () => {
           .catch(catchErrors(done));
       });
 
+      it('should create a random password when none is provided', done => {
+        users
+          .newUser(
+            {
+              username: 'someone',
+              email: 'someone@mailer.com',
+              role: 'admin',
+            },
+            domain
+          )
+          .then(() => {
+            expect(random.default).toHaveBeenCalled();
```

Comment:
Since you are mocking the random object, it would be great to expect that a User is created, and has the mock value 'mypass'

Reply:
Okay. On it

Reply:
The user returned does not include the password. I will just check whether the user is actually created

Reply:
Mongodb does not return the user password by default, you have to 'select' it on your query to mongo.

Reply:
Aah okay

Reply:
Done @konzz